mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExSamples.Mixfile do
     [
       app: :exsamples,
       version: "0.1.0",
-      elixir: "~> 1.13.2",
+      elixir: "~> 1.13 and >= 1.13.2",
```

Comment:
@msaraiva Not sure if it's the right way to do.

Reply:
I believe that will work just fine. Thanks @Malian!